lib/graphql_rails/attributes/input_type_parser.rb

```diff
@@ -59,9 +59,10 @@ def dynamicly_defined_type
 
       def parsed_list_type
         list_type = parsed_inner_type.to_list_type
+        list_type = list_type.to_graphql if list_type.respond_to?(:to_graphql)
```

Comment:
this should go under `if` part otherwise it will affect 67 line too (non required list case)